Library/Homebrew/formula_auditor.rb

```diff
@@ -560,9 +560,11 @@ def get_repo_data(regex)
       [user, repo]
     end
 
-    def audit_specs
+    def audit_head_only
       problem "Head-only (no stable download)" if head_only?(formula)
```

Comment:
I'd actually be fine with moving this below a `next unless @core_tap` but before a `next unless @new_formula` so it only ever applies for homebrew/core. Thoughts?

Reply:
even better - sure its a good convention for core - but most non core stuff is .... lets say more bleeding edge.

I think we should be careful to allow one to re-enable the head only checkin they want to follow the convention?

Reply:
@rickmark Nah, I think it's fine to just make it core-only, thanks!